cloudant-client/src/main/java/com/cloudant/client/api/model/PartitionInfo.java

```diff
@@ -21,6 +21,70 @@
  */
 public class PartitionInfo {
 
+    public static class Indexes {
```

Comment:
I think this belongs in `DbInfo` rather than `PartitionInfo`

Reply:
Ah, yes. I updated the wrong model here. Copied/pasted the new defs over to `DbInfo`.